wallet/res/xml/preference_headers.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<preference-headers xmlns:android="http://schemas.android.com/apk/res/android" >
-
-    <header
-        android:fragment="de.schildbach.wallet.ui.preference.DiagnosticsFragment"
-        android:title="@string/preferences_category_diagnostics" />
-    <header
-        android:fragment="de.schildbach.wallet.ui.preference.AboutFragment"
-        android:title="@string/about_title" />
-
-</preference-headers>
```

Comment:
Doesn't look like these fragments are used now anywhere.

Reply:
This is from the version 6 settings screens.